client/js/components/navbar.jsx

```diff
@@ -23,7 +35,12 @@ class Navbar extends React.Component {
   }
 
   render() {
-    let iconElementRight = (<RaisedButton style={{marginTop: '6px'}} onClick={ () => this.refs.dialog.open()} label="Add new snippet" primary={true}/>);
+    let iconElementRight = '';
```

Comment:
you don't need to assign value at declaration. `let iconElementRight;` works fine